src/fluent-bit.c

```diff
@@ -1036,6 +1036,9 @@ int flb_main(int argc, char **argv)
         { "http_port",       required_argument, NULL, 'P' },
 #endif
         { "enable-hot-reload",     no_argument, NULL, 'Y' },
+#ifdef FLB_SYSTEM_WINDOWS
+        { "maxstdio",        required_argument, NULL, 'M' },
```

Comment:
prefix is missing

Reply:
Fixed in https://github.com/fluent/fluent-bit/pull/9707/commits/beff608dc69d81aab78a03eae041b25438783efe.